make/addons.mk

```diff
@@ -87,8 +87,7 @@ template-helm-repository: generate-mindthegap-repofile ## this is used by goreal
 
 .PHONY: list-images
 list-images:
-	cd hack/tools/fetch-images && go build
-	./hack/tools/fetch-images/fetch-images \
-		-chart-directory=./charts/cluster-api-runtime-extensions-nutanix/ \
-		-helm-chart-configmap=./charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml \
-		-caren-version=$(CAREN_VERSION) >> caren-images.txt
+	cd hack/tools/fetch-images && go run . \
```

Comment:
This will also print log messages about downloading packages in clean environment. i.e. the release job in GHA may print logs similar to: 
```
❯ make list-images
go: downloading sigs.k8s.io/cluster-api v1.8.1
go: downloading k8s.io/cli-runtime v0.30.3
go: downloading k8s.io/client-go v0.30.3
go: downloading k8s.io/apiextensions-apiserver v0.30.3
go: downloading go4.org v0.0.0-20201209231011-d4a079459e60
go: downloading k8s.io/kubectl v0.30.3
go: downloading k8s.io/apiserver v0.30.3
go: downloading google.golang.org/grpc v1.62.2
go: downloading go.opentelemetry.io/otel v1.24.0
go: downloading go.opentelemetry.io/otel/trace v1.24.0
go: downloading go.opentelemetry.io/otel/metric v1.24.0
go: downloading k8s.io/component-base v0.30.3
docker.io/nutanix/ntnx-csi-precheck:3.0.0
```
should we redirect stderr to `/dev/null` when running `go run`

Reply:
I'll change back to `go build` and calling the built executable directly so as not to hide build issues.

Reply:
Actually even using `go build` will print out the same log messages but because they're going to stderr I think this is fine. In GHA that won't matter because it will only be used when releasing, in which case the stdout is redirected to a `caren-images.txt`.